scripts/who_is_better.coffee

```diff
@@ -76,30 +87,24 @@ module.exports = (robot) ->
   #     res.status(500).json({text: 'Я занят, попросите позже'})
   #   )
 
-
-
-class Pretty
+class QuiryAPI
   TTL: 5
-
+  API_URL: 'http://localhost:3000/export_to_jarvis'
```

Comment:
env variables.
for example `API_URL: "#{process.env.KODER_TIME_TRACKER_API}/export_to_jarvis"`